weblate/trans/models/change.py

```diff
@@ -28,15 +28,28 @@
 from weblate.utils.state import StringState
 
 if TYPE_CHECKING:
-    from datetime import datetime
-
     from weblate.auth.models import User
     from weblate.trans.models import Translation
 
 
 CHANGE_PROJECT_LOOKUP_KEY = "change:project-lookup"
 
 
+def dt_as_day_range(dt: datetime | date) -> tuple[datetime]:
```

Comment:
The type annotation is wrong, it returns two datetimes:

```suggestion
def dt_as_day_range(dt: datetime | date) -> tuple[datetime, datetime]:
```